README.md

```diff
@@ -262,7 +262,6 @@ Or select one of the playlists from the list below.
 		<tr><td align="left">🇸🇰&nbsp;Slovakia</td><td align="right">30</td><td align="left" nowrap><code>https://iptv-org.github.io/iptv/countries/sk.m3u</code></td><td align="left"></td></tr>
 		<tr><td align="left">🇸🇮&nbsp;Slovenia</td><td align="right">6</td><td align="left" nowrap><code>https://iptv-org.github.io/iptv/countries/si.m3u</code></td><td align="left"></td></tr>
 		<tr><td align="left">🇸🇴&nbsp;Somalia</td><td align="right">9</td><td align="left" nowrap><code>https://iptv-org.github.io/iptv/countries/so.m3u</code></td><td align="left"></td></tr>
-		<tr><td align="left">🇿🇦&nbsp;South Africa</td><td align="right">2</td><td align="left" nowrap><code>https://iptv-org.github.io/iptv/countries/za.m3u</code></td><td align="left"><code>http://i.mjh.nz/za/DStv/epg.xml.gz</code></td></tr>
```

Comment:
Why are you removing South Africa?

Reply:
Because there is no South African TV stations in the channels.

Reply:
The country code you provided is incorrect.

Reply:
@zo-shin

1st: All country codes are taken from here: https://en.wikipedia.org/wiki/ISO_3166-2
2nd: This file is automatically generated and all changes in it will be erased during the next update.
3rd: Before making any changes, please read this guide first: https://github.com/iptv-org/iptv/blob/master/.github/CONTRIBUTING.md